docs/installation/installation_google_cloud_kubernetes.md

```diff
@@ -0,0 +1,47 @@
+## Installing Yacy on Google Cloud with Docker

```

Comment:
@nikhilrayaprolu this is not kubernetes. This is Google Compute Engine. Update the file name and title accordingly.